src/__experimental__/components/field-help/field-help.component.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FieldHelpStyle from './field-help.style';
+
+/**
+ * To import FieldHelp:
+ *
+ *   import FieldHelp from 'carbon-react/lib/__experimental__/components/field-help';
+ *
+ * To render FieldHelp:
+ *
+ *   <FieldHelp>This provides more information for the field.</FieldHelp>
+ */
```

Comment:
We're tending towards deleting these documentation comments from components as we're adding them in on Storybook. Don't know if we're taking that approach for the `experimental` components?

Reply:
Are we? Where are they being stored instead? There is some benefit in keeping them for IDE purposes, for example VSCode will show these comments for a given class

![Screenshot 2019-03-14 at 09 15 48](https://user-images.githubusercontent.com/1668080/54344893-cb55f880-4639-11e9-86cd-2c1d9eda641e.png)

Reply:
Though to be honest the import info is probably less relevant considering you've already imported it!

Reply:
I think that's what we were thinking with a lot of those docs when we were looking at them last week. It's rare that there's any really useful information in there.

Where there has been, we've been adding them to Storybook using the Info AddOn. @stevebutler2210 has done a load of work styling them too...

![Screenshot 2019-03-14 at 09 27 59](https://user-images.githubusercontent.com/4471057/54345711-816e1200-463b-11e9-98e5-bddaa2425a77.png)